src/System.Diagnostics.PerformanceCounter/src/System/Diagnostics/CounterSampleCalculator.cs

```diff
@@ -236,7 +236,7 @@ private static void LoadPerfCounterDll()
 
             string installPath = SharedUtils.GetLatestBuildDllDirectory(".");
 
-            string perfcounterPath = Path.Combine(installPath, "perfcounter.dll");
+            string perfcounterPath = Path.Combine(installPath, "perfcounter.dll".ToLowerInvariant());
```

Comment:
Why does this fix this?

Reply:
The way I repro the bug is. Build my corefx repo and the test project. Copy the bin folder to the Spanish machine. and run the tests using Xunit.
The reason for the failure could be that the string "perfcounter.dll"  is stored somewhat differently on the Spanish machine. I will investigate more and update why and how it is fixing it

Reply:
I'm a little surprised at this as well.  I was wondering where the incorrectly localized string was that caused this, but I don't see one so far.  Do you know why machine name "." is OK on English, but not on non-English?

Reply:
Updated the descrption with more investigation results